src/test/java/de/tum/in/www1/artemis/tutorialgroups/TutorialGroupIntegrationTest.java

```diff
@@ -308,4 +355,54 @@ void registerMultipleStudents_asInstructor_shouldRegisterStudents() throws Excep
         assertThat(notFoundStudents).containsExactly(studentNotInCourse);
     }
 
+    private void verifyPrivateInformationIsHidden(TutorialGroup tutorialGroup) {
+        assertThat(tutorialGroup.getRegistrations()).isEqualTo(Set.of());
+        assertThat(tutorialGroup.getTeachingAssistant()).isEqualTo(null);
+        assertThat(tutorialGroup.getCourse()).isEqualTo(null);
+    }
+
+    private void verifyPrivateInformationIsShown(TutorialGroup tutorialGroup, Integer numberOfRegistrations) {
+        assertThat(tutorialGroup.getRegistrations()).hasSize(numberOfRegistrations);
+        assertThat(tutorialGroup.getTeachingAssistant()).isNotNull();
+        assertThat(tutorialGroup.getCourse()).isNotNull();
+    }
+
+    private void getOneOfCoursePrivateInfoHiddenTest() throws Exception {
```

Comment:
Maybe setOne... or createOne... instead?

Reply:
Just named it now: oneOfCoursePrivateInfoHiddenTest. I hope thats ok?